### DISCUSSION TITLE
Allow users with email registration methods to change their address.

### DISCUSSION THREAD

**Discussion by ghost** [civitai/civitai#573] [Ideas]
You allow email as a registration method since December 2022 but provide no option to change that email address, and your "Support Us" membership/donate links through Stripe use whichever email address is present with no way to modify it. There's nothing in the terms of service prohibiting temporary domains, nor was there when I joined. Additionally on Thursday, you started blocking ~3500 temporary email domains and no longer send login emails to them.

I understand the need to control spam or bot users, but frankly I first signed up with a temporary email address because I wasn't sure if I'd want to stay. I've since connected other accounts, but if I hadn't I'd now be locked out. I also can't buy a membership or donate even if I may be inclined to, because the Stripe links are tied to that now-useless address. Obviously I don't know the number of legitimate users you've just dropped, but after 6 months of email login availability, it isn't clear to me why it isn't exposed to the account settings page to adjust.

**Comment by ghost** [comment]
Coming up on 4 months later, an email newsletter has been introduced and yet still no way to set or change an email address. I can't even become a supporter of the site to try to incentivize such a feature, because the stripe page is tied to the useless email address.

**Comment by ghost** [comment]
Closing, no engagement and clearly not a priority.

**Comment by Latentnaut** [comment]
I also need an email change.
